src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String welcomeMessage = "Hello! I'm KJ\nHow can I help you today?";
+
+    private static final String completeMessage = "Nice! I've marked this task as done:";
+
+    private static final String byeMessage = "Bye. Hope to see you again soon!";
+
+    private static final String listMessage = "Here are the tasks in your list:";
+
+    private static final int MAXIMUM_TASKS = 100;
+
+    private static boolean shouldContinue;
+
+    private static String command;
+
+    private static String action;
+
+    private static Scanner input;
+
+    private static Task[] instructions;
+
+    private static int count;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        initDuke();
+        displayWelcome();
+        while (shouldContinue) {
+            readInput();
+            setAction();
+            processCommand();
+        }
+    }
+
+    private static void processCommand() {
+        switch(action) {
+        case "bye":
+            sayBye();
+            break;
+        case "list":
+            listTasks();
+            break;
+        case "done":
+            completeTask();
+            break;
+        case "deadline":
+            setDeadline();
+            confirmTask();
+            break;
+        case "event":
+            createEvent();
+            confirmTask();
+            break;
+        case "todo":
+            addTodo();
+            confirmTask();
+            break;
```

Comment:
Consider having a "default" case to handle any events that might not correspond with the above cases. These can include typos made by the user or invalid commands entered

Reply:
That's very true, I have not accounted for any cases where the user types in invalid input. Will make sure to add this next